packages/confirm-dialog/src/vaadin-confirm-dialog-overlay.js

```diff
@@ -83,7 +84,7 @@ defineCustomElement(ConfirmDialogOverlay);
  * @private
  */
 class ConfirmDialogDialog extends ConfirmDialogBaseMixin(
-  DialogBaseMixin(OverlayClassMixin(ThemePropertyMixin(PolylitMixin(LitElement)))),
+  DialogBaseMixin(OverlayClassMixin(ThemePropertyMixin(CSSInjectionMixin(PolylitMixin(LitElement))))),
```

Comment:
In theory this shouldn't be needed as the internal dialog isn't supposed to be themable.